db/sql/sql.go

```diff
@@ -1274,6 +1282,12 @@ func addToFilter(s *schema.Schema, q sq.SelectBuilder, filter interface{}, join
 
 			// TODO: add other operators
 			value := filter["value"]
+			substr, ok := value.(search.Search)
+			if ok {
+				sqlizer = append(sqlizer, sq.Like{column: substr.Value})
```

Comment:
there are not tests for it (and it looks like it breaks the API). please see how the "extended" API looks like and add tests for it [this](https://github.com/cloudwan/gohan/blob/master/db/sql/sql_test.go#L425) block

Reply:
except for injecting the squirrel.Like in some of the already existing test cases couldn't find any need for new test cases...